shared/fs/destination-picker/index.js

```diff
@@ -116,9 +116,14 @@ export default (isMobile
           <Kb.Box2 direction="vertical" centerChildren={true} style={styles.mobileHeaderContent}>
             <Kb.Box2 direction="horizontal" centerChildren={true} gap="xtiny">
               <FsCommon.PathItemIcon size={12} path={Types.pathConcat(props.parentPath, props.targetName)} />
-              <Kb.Text type="BodySmallSemibold" lineClamp={1}>
-                {props.targetName}
-              </Kb.Text>
+              <Kb.Box2 direction="horizontal">
+                <Kb.Text type="BodySmallSemibold" lineClamp={1}>
+                  {props.targetNameWithoutExtension}
+                </Kb.Text>
+                <Kb.Text type="BodySmallSemibold">
+                  {props.targetExtension}
+                </Kb.Text>
+              </Kb.Box2>
```

Comment:
Is the extra box necessary?

Reply:
Maybe, maybe not. I believe I tried it without and it didn't work visually. But my build is currently not working due to "new.target can't come after a prefix operator."...

Reply:
OK yeah it's necessary, the cells don't collapse together without the `Box2`

Reply:
Oh I see; I guess it's because of the gap attr in outter box. Sounds good!